app/request_handler.py

```diff
@@ -226,13 +226,20 @@ def pruneEvents():
 
     for event in eventDetails:
         location = event.key()
+        if ("localEndTime" not in event.val()):
+            deleteEventByLocation(location)
+            continue
+
         endTime = parser.parse(event.val().get("localEndTime"))
         if endTime < cutoff.replace(tzinfo=None):
-            db.child(eventsTable).update({
-                "details/" + location: None,
-                "cache/" + location: None,
-                "locations/" + location: None
-            })
+            deleteEventByLocation(location)
+
+def deleteEventByLocation(location):
```

Comment:
My understanding is that this is not location, but by event key (as in time+location). Is this correct?